echidna/core/fill_spectrum.py

```diff
@@ -1,3 +1,4 @@
+import root_help
```

Comment:
Ah OK, so it is imported where rat or ROOT is imported, before their imports, and not in the module that parses the arguments.